DigitalLearningSolutions.Data/DataServices/CentresDataService.cs

```diff
@@ -1,150 +1,169 @@
+                            c.BannerText,
+                            c.SignatureImage,
+                            c.CentreLogo,
+                            c.ContactForename,
+                            c.ContactSurname,
+                            c.ContactEmail,
+                            c.ContactTelephone,
+                            c.pwTelephone AS CentreTelephone,
+                            c.pwEmail AS CentreEmail,
+                            c.pwPostCode AS CentrePostcode,
+                            c.pwHours AS OpeningHours,
+                            c.pwWebURL AS CentreWebAddress,
+                            c.pwTrustsCovered AS OrganisationsCovered,
+                            c.pwTrainingLocations AS TrainingVenues,
+                            c.pwGeneralInfo AS OtherInformation
+                        FROM Centres AS c
+                        INNER JOIN Regions AS r ON r.RegionID = c.RegionID
+                        WHERE CentreID = @centreId",
+                new { centreId }
+            );
+
+            if (centre == null)
+            {
+                logger.LogWarning($"No centre found for centre id {centreId}");
+                return null;
+            }
+
+            if (centre.CentreLogo?.Length < 10)
+            {
+                centre.CentreLogo = null;
+            }
+
+            return centre;
+        }
+
+        public void UpdateCentreManagerDetails
+        (
+            int centreId,
+            string firstName,
+            string lastName,
+            string email,
+            string? telephone
+        )
+        {
+            connection.Execute(
+                @"UPDATE Centres SET
+                    ContactForename = @firstName,
+                    ContactSurname = @lastName,
+                    ContactEmail = @email,
+                    ContactTelephone = @telephone
+                WHERE CentreId = @centreId",
+                new { firstName, lastName, email, telephone, centreId }
+            );
+        }
+
+        public (string firstName, string lastName, string email) GetCentreManagerDetails(int centreId)
+        {
+            var info = connection.QueryFirstOrDefault<(string, string, string)>(
+                @"SELECT ContactForename, ContactSurname, ContactEmail
+                        FROM Centres
+                        WHERE CentreID = @centreId",
+                new { centreId }
+            );
+            return info;
+        }
+
+        public string[] GetCentreIPPrefix(int centreId)
+        {
+            var IPPrefix = connection.QueryFirstOrDefault<string?>(
+                @"SELECT IPPrefix
+                        FROM Centres
+                        WHERE CentreID = @centreId",
+                new { centreId }
+            );
+
+            var IPPrefixes = IPPrefix?.Split(',', StringSplitOptions.RemoveEmptyEntries);
+            IPPrefixes ??= new string[0];
+
+            return IPPrefixes;

```

Comment:
Could we do return `IPPrefixes ?? new string[0];` instead of overriding the IPPrefixes value (it's a bit more concise)